Resources/translations/SonataAdminBundle.fr.xliff

```diff
@@ -86,6 +86,10 @@
                 <source>title_create</source>
                 <target>Créer</target>
             </trans-unit>
+            <trans-unit id="title_show">
+                <source>title_show</source>
+                <target>title_show</target>
```

Comment:
`Afficher "%name%"` is good, `Voir "%name%"` would be more natural IMO. Or maybe just "%name%" would be enough? What is currently displayed?